examples/enums/enums.bal

```diff
@@ -0,0 +1,53 @@
+import ballerina/io;
+
+// Enums are defined using `enum` keyword.
+enum Color {
+  RED,
+  GREEN,
+  BLUE
+}
+
+// The value of an enum member can be overridden in the enum declaration.
```

Comment:
Shall we mention the constraint that it has to be a string? Also instead of overridden I think we can say something like
```suggestion
// An enum member can explicitly specify a value.
```